components/pagination/stories/template.js

```diff
@@ -48,6 +47,16 @@ export const Template = ({
 				})}
 			</nav>
 		`;
+	} else if (variant == "button") {
+		return SplitButton({
+			position: "left",
+			variant: "accent",
```

Comment:
So we can toggle the button style between the 3 variants, do we want to also add variant to the controls for pagination?

Reply:
It already exists as a disabled control. I'm not sure it should be enabled since these pagination types are so different and not really variation classes? The button one is just a SplitButton but with two classes added to the buttons.